shared/routes/Index.html

```diff
@@ -0,0 +1,169 @@
+<div class='hero'>
+	<div class='left'>
+		<h1>Svelte</h1>
+	</div>
+
+	<div class='right'>
+		<h2>The magical disappearing UI framework</h2>
+
+		<ul>
+			<li><strong>The web's obesity crisis, solved.</strong> Svelte turns your templates into tiny, framework-less vanilla JavaScript.</li>
+			<li><strong>Simple and familiar.</strong> Build apps out of composable, easy-to-write blocks using languages you already know.</li>
+			<li><strong>Stupid fast, rock solid.</strong> Compile-time static analysis ensures the browser does no more work than it needs to.</li>
+		</ul>
+
+		<a class='learn-svelte' href='/guide'>Learn Svelte</a>
+	</div>
+</div>
+
+<style>
+	.hero {
+		position: relative;
+		padding: 2em 1em;
+		/*background-color: #333;
+		color: #f4f4f4;*/
+		background-color: white;
+		color: #333;
+		clear: fix;
+		min-height: calc(100vh - 7em);
+		box-sizing: border-box;
+	}
+
+	ul {
+		max-width: 32em;
+		padding: 0;
+		margin: 0 0 3em 0;
+	}
+
+	li {
+		position: relative;
+		list-style: none;
+		margin: 0 0 1em 0;
+		font-family: Rajdhani, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+		color: #666;
+	}
+
+	li::before {
+		position: absolute;
+		content: '◆';
+		top: 0em;
+		left: -1.4em;
+		font-size: 0.8em;
+		/*color: rgba(255,255,255,0.4);*/
+		color: #ddd;
+	}
+
+	strong {
+		font-family: Rajdhani, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+		margin: 0 auto;
+		color: #333;
+	}
+
+	.left {
+		padding: 0 0 0 1em;
+	}
+
+	h1 {
+		display: block;
+		font-family: Rajdhani, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+		color: rgb(170,30,30);
+		font-size: 30vw;
+		font-weight: 300;
+		line-height: 1;
+		margin: 0;
+		text-transform: lowercase;
+		padding: 0.5em 0 0 0;
+	}
+
+	.right {
+		padding: 0 0 0 1em;
+		margin: 0 0 2em 0;
+	}
+
+	h2 {
+		font-size: 7.2vw;
+		font-family: Rajdhani, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+		font-weight: 300;
+		display: block;
+		margin: 0 0 4rem 0;
+		max-width: 11em;
+	}
+
+	.learn-svelte {
+		background-color: rgb(170,30,30);
+		color: white;
+		padding: 0.5em 2em;
+		border-radius: 0.2em;
+		line-height: 1;
+		font-family: Rajdhani, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+		font-size: 1.2em;
+		font-weight: 800;
+		text-transform: uppercase;
+		width: calc(100% - 1em);
+		max-width: 16em;
+		display: block;
+		text-align: center;
+	}
+
+	.learn-svelte:hover {
+		background-color: rgba(170,30,30,0.9);
+	}
+
+	@media (min-width: 480px) {
+		.left, .right {
+			max-width: 32em;
+			margin: 0 auto;
```

Comment:
You need to make it a block element (or take care to not add those classes to inline elements as that won't have an effect).